dcp_prototype/backend/wrangling/migrations/scripts/copy_data_files.py

```diff
@@ -0,0 +1,133 @@
+import sys
+
+import boto3
+import concurrent.futures
+import json
+import os
+import typing
+
+from argparse import ArgumentParser
+
+from boto3.s3.transfer import TransferConfig
+sys.path.insert(0, "")  # noqa
+from dcp_prototype.backend.wrangling.migrations.utils.util import list_files_in_bucket
+
+s3 = boto3.resource('s3')
+SOURCE_BUCKET = 'org-hca-dss-prod'
+DESTINATION_BUCKET = 'dunitz-prod-copy'
```

Comment:
If possible, could we change the default to be a bucket that doesn't contain your username? Probably best to keep it repeatable by others.